x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-sqs-overview.json

```diff
@@ -5,111 +5,10 @@
         "description": "Overview of AWS SQS Metrics",
         "hits": 0,
         "kibanaSavedObjectMeta": {
-          "searchSourceJSON": {
-            "filter": [],
-            "query": {
-              "language": "kuery",
-              "query": ""
-            }
-          }
+          "searchSourceJSON": "{\"filter\":[],\"query\":{\"language\":\"kuery\",\"query\":\"\"}}"
```

Comment:
This seems like the offending line :confused:

Reply:
yeah turned out I was exporting dashboard without decoding them.